ansible/configs/openshift-demos/msa-cicd-eap-full.yml

```diff
@@ -61,3 +61,58 @@
         gogs_hostname: "gogs-{{ project_cicd }}.{{ apps_hostname_suffix }}"
         prune_deployments_selector_prod: ""
         prune_deployments_selector_stage: "comp-required!=true,app!=inventory"
+
+    # verify deployments in ci/cd project
+    - name: wait for deployments in project {{ project_cicd }} to complete (succeed or fail)
+      shell: "oc get pods -n {{ project_cicd }} | grep '\\-deploy' | grep 'Running'"
```

Comment:
I would rather take advantage of json output here. `oc` can output json. and then use jinja2 filters like `from_json` and `json_query` instead of grep

```
command: "oc get pods -n {{ project_cicd }} -o json"```